Oxide.Rust/RustCore.cs

```diff
@@ -149,14 +150,32 @@ private void OnServerInitialized()
                 Interface.Oxide.LogWarning("The server is currently listed under Community. Please be aware that Facepunch only allows admin tools" +
                                            "(that do not affect gameplay) under the Community section");
 
+            // Custom save timer if server save interval is high (prevents data from never being saved if server crashes)
+            if (ConVar.Server.saveinterval > 3600)
```

Comment:
The "never" comment here is a bit misleading, as the server could easily crash and lose any data that was lost before 600 seconds. I don't think this is the ideal method to handle this either. It'd be better to just rely on what the server has set for the interval if it has one, otherwise we'd set one for games that don't have this.